color_cx: 313.90618896484375
color_cy: 246.54579162597656
color_fx: 616.7605590820312
color_fy: 617.09521484375

depth_cx: 316.8928527832031
depth_cy: 242.30789184570312
depth_fx: 379.6465148925781
depth_fy: 379.6465148925781

# transformation from velodyne to camera_link
rpy_lc: [0, 0.349066, 0] # roll, pitch yall
t_lc: [-0.15, 0, 0.95]


# tranformation from camera_link to color/depth optical frame
rpy_c2co: [-1.572, -0.002, -1.571]
t_c2co: [-0.001, 0.015, -0.000]

rpy_c2do: [-1.571, -0.000, -1.571]
t_c2do: [0.000, 0.000, 0.000]